models/varGlobales.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
We should name this file GlobalVars

Reply:
What about "city" ? 
Like that, requesting a value from global storage would look something like city.inventory.water.
We can work on access prototypes later ^^

Reply:
Seems great